modules/migration-converting-the-storage-class-of-persistent-volumes-cam.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/advanced-migration-options-mtc.adoc
+
+:_content-type: PROCEDURE
+[id="migration-converting-the-storage-class-of-persistent-volumes-cam_{context}"]
+= Converting the storage class of persistent volumes in the {mtc-short} web console
+
+You can convert the storage class of persistent volumes (PVs) by migrating them within the same cluster. For that, you must create and run a migration plan in the {mtc-full} ({mtc-short}) web console.
+
+.Prerequisites
+
+* You must be logged in as a user with `cluster-admin` privileges on the cluster on which {mtc-short} is running.
+* You must add the cluster to the {mtc-short} web console.
+
+.Procedure
+
+. In the left-side navigation pane of the web console, click *Projects*.
+. In the list of projects, click your project.
++
+The *Project details* page opens.
+. Click the DeploymentConfig. Note the name of its running pod.
```

Comment:
> Click the DeploymentConfig.

Suggestion: Click the DeploymentConfig button.

Avoid using UI elements as words.

Reply:
Changed